### PR TITLE
fix: enable CI on release bump commit

### PR DESCRIPTION
**Pull request by behnazh**

### CODE REVIEW

File: .github/workflows/build.yaml [hunk 1]
```diff
@@ -71,7 +76,7 @@ jobs:
     - name: Install dependencies
       run: make setup
     - name: Create changelog and bump
-      if: github.ref == 'refs/heads/main' && matrix.os == env.ARTIFACT_OS && matrix.python == env.ARTIFACT_PYTHON
+      if: ${{ matrix.os == 'ubuntu-latest' && matrix.python == '3.10' && !startsWith(github.event.commits[0].message, 'bump:') }}
```
Comment (jenstroeger) [inline .github/workflows/build.yaml:79]: Dang! I thought that `if:` interprets its value as JS?

**Review by jenstroeger — APPROVED** [review]

**Comment by jenstroeger** [comment]
Also, I think we should turn

https://github.com/jenstroeger/python-package-template/blob/c7541754294c634cd807068ffaf37bd0279cbb52/.github/workflows/build.yaml#L84

into
```
cz bump --yes 
```
because on the very first bump no tags exist yet and the Action would fail:
```
> cz bump
Tag v0.0.0 could not be found. 
Possible causes:
- version in configuration is not the current version
- tag_format is missing, check them using 'git tag --list'

? Is this the first tag created? (Y/n)                                                                               
```

File: .github/workflows/build.yaml (hunk 1 above)
Comment (behnazh) [reply .github/workflows/build.yaml]: > Dang! I thought that `if:` interprets its value as JS?

It does, but notice the `!` which is a reserved symbol in YAML, hence using `${{}}`.

Comment (behnazh) [reply .github/workflows/build.yaml]: > And you wired the OS and Python, instead of using the variables — only here, and not in other lines?

:facepalm:

Comment (jenstroeger) [reply .github/workflows/build.yaml]: Why do I keep asking the same question 🤦🏻‍♂️

**Comment by jenstroeger** [comment]
And as a follow-up, check this out issue https://github.com/commitizen-tools/commitizen/issues/549 😳

**Review by jenstroeger — APPROVED** [review]

**Pull request merged**